packages/api/src/tests/elastic-spec.js

```diff
@@ -1,483 +0,0 @@
-import test from 'tape-promise/tape'
```

Comment:
This test was only running Elasticsearch queries and didn't cover any gnomadjs code.